web-client/src/environments/environment.palau-prod.ts

```diff
@@ -28,19 +28,19 @@ export const environment: Environment = {
   // USDC from https://testnet.folks.finance/faucet
   defaultAlgorandAssetId: 67395862,
   xrplClient: {
-    server: 'wss://s1.cbdc-sandbox.rippletest.net:51233',
+    server: 'wss://s1.ripple.com',
```

Comment:
Is it not necessary to specify the port here?  I recall this creating issues for us in the past.
```suggestion
    server: 'wss://s1.ripple.com:51233',
```

Reply:
Evidently this server works without the port for web sockets, however it also works with the port so I'll keep it.